common/djangoapps/student/email_helpers.py

```diff
@@ -28,10 +28,15 @@ def generate_activation_email_context(user, registration):
         'key': registration.activation_key,
         'lms_url': configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
         'platform_name': configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
+        'contact_mailing_address': configuration_helpers.get_value(
+            'contact_mailing_address',
```

Comment:
`CONTACT_MAILING_ADDRESS`?

Reply:
@sarina, Actually, [this piece of code](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/ace_common/template_context.py#L37) has been used as a reference. There you can see the lower case `contact_mailing_address` is being used.